### PR TITLE
[Shrd] Update tokens labels (no metadata)

### PR DESCRIPTION
**Pull request by mvaivre**

### CODE REVIEW

File: apps/desktop-wallet/locales/id-ID/translation.json [hunk 1]
```diff
@@ -402,7 +402,7 @@
   "Unlocked at": "Tidak terkunci pada",
   "Unlocks at": "Buka kunci di",
   "Unselect all": "Batalkan semua pilihan",
-  "Unverified": "Tidak terverifikasi",
+  "No metadata": "Tidak ada metadata",
```
Comment (nop33) [inline apps/desktop-wallet/locales/id-ID/translation.json:405]: You checked this?

File: apps/desktop-wallet/locales/ru-RU/translation.json [hunk 2]
```diff
@@ -402,7 +402,7 @@
   "Unlocked at": "Разблокирован в",
   "Unlocks at": "Разблокируется в",
   "Unselect all": "Отменить выбор",
-  "Unverified": "Не проверено",
+  "No metadata": "Нет метаданных",
```
Comment (nop33) [inline apps/desktop-wallet/locales/ru-RU/translation.json:405]: Mika speaks Russian now 😆

File: apps/desktop-wallet/locales/vi-VN/translation.json [hunk 3]
```diff
@@ -400,7 +400,7 @@
   "Unlocked at": "Đã mở khoá vào",
   "Unlocks at": "Mở khoá vào",
   "Unselect all": "Bỏ chọn tất cả",
-  "Unverified": "Chưa được xác nhận",
+  "No metadata": "Không có metadata",
```
Comment (nop33) [inline apps/desktop-wallet/locales/vi-VN/translation.json:403]: and Vietnamese, damn!

File: apps/desktop-wallet/src/components/Tooltip.tsx [hunk 4]
```diff
@@ -26,5 +26,5 @@ export type HasTooltip<T> = T & { tooltip?: string }
 export default styled(Tooltip)`
   background-color: rgb(10, 10, 10);
   color: #fff;
-  z-index: 1;
+  z-index: 3;
```
Comment (nop33) [inline apps/desktop-wallet/src/components/Tooltip.tsx:29]: Why was this needed? This tells me there is something wrong with the stacking context...

File: apps/explorer/src/pages/AddressInfoPage/NFTDetailsModal.tsx [hunk 5]
```diff
@@ -99,7 +99,7 @@ const NFTDetailsModal = ({ nft, collection, ...props }: NFTDetailsModalProps) =>
                 )}
               </>
             ) : (
-              <CardStyled>{t('Incorrect metadata')}</CardStyled>
+              <CardStyled>{t('Wrong / old format')}</CardStyled>
```
Comment (nop33) [inline apps/explorer/src/pages/AddressInfoPage/NFTDetailsModal.tsx:102]: Is it a french thing to add spaces around a `/`? I'd be in favor of "Wrong/old format"

Comment (nop33) [reply apps/explorer/src/pages/AddressInfoPage/NFTDetailsModal.tsx:102]: Why is this not needed in our other apps and only in the explorer?

File: apps/desktop-wallet/src/components/Tooltip.tsx (hunk 4 above)
Comment (mvaivre) [reply apps/desktop-wallet/src/components/Tooltip.tsx:29]: Because of modal (z-index of 2), tooltips were shown below.

File: apps/explorer/src/pages/AddressInfoPage/NFTDetailsModal.tsx (hunk 5 above)
Comment (mvaivre) [reply apps/explorer/src/pages/AddressInfoPage/NFTDetailsModal.tsx:102]: I think I remember a past discussion about filtering wrong or old metadata format in our wallets, but show them for debugging purposes in the explorer 🤔

File: apps/desktop-wallet/src/components/Tooltip.tsx (hunk 4 above)
Comment (nop33) [reply apps/desktop-wallet/src/components/Tooltip.tsx:29]: I don't see any `z-index: 2` in a modal. I will have a look. I would like to not start re-introducing arbitrary z-index values and rather fix our stacking context.

**Review by nop33 — COMMENTED** [review]
I propose this change: https://github.com/alephium/alephium-frontend/pull/495

Also, in the light theme, the circle around the `i` icon is almost invisible

image


I don't have something else to add in this PR :) Good job! 👏
